Cargo.toml

```diff
@@ -1,24 +1,38 @@
 [workspace]
 resolver = "2"
+
 members = [
 	"lofty",
 	"lofty_attr",
 	"ogg_pager",
 	"fuzz",
 ]
 
+[workspace.package]
+rust-version = "1.82"
+
 [workspace.dependencies]
 byteorder = "1.5.0"
 
 [workspace.lints.rust]
 missing_docs                   = "deny"
 rust_2018_idioms               = { level = "deny", priority = -1 }
+rust_2021_compatibility        = { level = "deny", priority = -1 }
+rust_2024_compatibility        = { level = "deny", priority = -1 }
 trivial_casts                  = "deny"
 trivial_numeric_casts          = "deny"
 unused_import_braces           = "deny"
 explicit_outlives_requirements = "deny"
 unknown_lints                  = "allow"
 
+# Opt-in to new drop order of Edition 2024.
+# TODO: Remove if no longer needed.
+tail_expr_drop_order           = "allow"
+if_let_rescope                 = "allow"
+
+# TODO: Remove after switching to Edition 2024.
+edition_2024_expr_fragment_specifier = "allow"
```

Comment:
Are these still needed?

Reply:
Yes. Only after switching to `edition = "2024` both `if_let_rescope` and `edition_2024_expr_fragment_specifier` will probably become obsolete.

Reply:
You can bump to 2024 edition in this PR if you want. I don't think it's a breaking change.

Reply:
Switching to Edition 2024 requires to reformat many source files.

I will prepare an intermediate step with some workspace cleanup as part of this PR. Switching editions should be done separately.